src/Microsoft.Health.Fhir.Liquid.Converter/Constants.cs

```diff
@@ -9,5 +9,6 @@ public static class Constants
     {
         public const string Hl7v2DataKey = "hl7v2Data";
```

Comment:
Curiously, do we intend to use "msg" for all resource types except v2?

Reply:
What about adding ```msg```  key support for hl7v2 and keep compatible with the original "hl7v2Data". So old clients can keep using the "hl7v2Data" but new clients should use the consistent "msg" key.

Reply:
After offline discussion, we decide not to change the key for v2 as it is widely used in templates and it's difficult to change the existing templates.